simplipy/system/__init__.py

```diff
@@ -272,6 +281,19 @@ def _coerce_state_from_raw_value(self, value: Union[str, None]) -> SystemStates:
             _LOGGER.error("Unknown system state: %s", value)
             return SystemStates.unknown
 
+    def _get_cameras_doorbells(self) -> List[Camera]:
```

Comment:
You only use this method once – why not just put its logic directly into the `cameras` property?